specification/logic/resource-manager/Microsoft.Logic/stable/2019-05-01/logic.json

```diff
@@ -5603,805 +7169,1033 @@
-          "type": "string",
-          "format": "date-time",
-          "readOnly": true,
-          "description": "Gets the end time."
+        "connectionParameters": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "jtoken"
+          },
+          "description": "The connection parameters."
         },
-        "status": {
-          "$ref": "#/definitions/WorkflowStatus",
-          "readOnly": true,
-          "description": "Gets the status."
+        "metadata" : {
+          "$ref": "#/definitions/ApiResourceMetadata",
+          "description": "The metadata."
         },
-        "code": {
-          "type": "string",
-          "readOnly": true,
-          "description": "Gets the code."
+        "runtimeUrls" : {
+          "type" : "array",
+          "description" : "The runtime urls.",
+          "items" : {
+            "type" : "string"
+          }
         },
-        "error": {
-          "$ref": "#/definitions/Object",
-          "readOnly": true,
-          "description": "Gets the error."
+        "generalInformation" : {
+          "$ref": "#/definitions/ApiResourceGeneralInformation",
+          "description": "The api general information."
         },
-        "trackingId": {
-          "type": "string",
-          "readOnly": true,
-          "description": "Gets the tracking id."
+        "capabilities" : {
+          "type" : "array",
+          "description" : "The capabilities.",
+          "items" : {
+            "type" : "string"
+          }
         },
-        "correlation": {
-          "$ref": "#/definitions/Correlation",
-          "description": "The correlation properties."
+        "backendService" : {
+          "$ref":"#/definitions/ApiResourceBackendService",
+          "description": "The backend service."
         },
-        "inputsLink": {
-          "$ref": "#/definitions/ContentLink",
-          "readOnly": true,
-          "description": "Gets the link to inputs."
+        "policies" : {
+          "$ref" :  "#definitions/ApiResourcePolicies",
+          "description" : "The policies for the API."
         },
-        "outputsLink": {
-          "$ref": "#/definitions/ContentLink",
-          "readOnly": true,
-          "description": "Gets the link to outputs."
+        "apiDefinitionUrl" : {
+          "type" : "string",
+          "description" : "The API definition."
         },
-        "trackedProperties": {
-          "$ref": "#/definitions/Object",
-          "readOnly": true,
-          "description": "Gets the tracked properties."
+        "apiDefinitions" : {
+          "$ref" : "#definitions/ApiResourceDefinitions",
+          "description" : "The api definitions."            
         },
-        "retryHistory": {
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/RetryHistory"
-          },
-          "description": "Gets the retry histories."
+        "integrationServiceEnvironment" :{
```

Comment:
why is this needed? ManagedAPI is already under an ISE.

Reply:
Its because the same object model is already used in shared environment.

Reply:
not sure I follow. If I am doing a GET on /integratedServiceEnvironments/ise1/managedApis/api1. Then why should the response of  api1 include the payload of ise1?

Reply:
Is there any contract which dictates that RP cannot return this? The object just contains a reference of the ISE it is present in.

Reply:
As discussed closing this.